Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -662,41 +662,45 @@ def url_report_output(response_json, url):
+        **Final URL**: {last_analysis.get("final_url")}
+        **Availability status**: {last_analysis.get("availability_status")}
+        **Domain**: {last_analysis.get("domain")}
+        **Serving IP Address**: {last_analysis.get("serving_ip_address")}\n ### Statistics\n **KNOWN**: {statistics.get(
+            "known")}
+        **SUSPICIOUS**: {statistics.get("suspicious")}
+        **MALICIOUS**: {statistics.get("malicious")}
+        **UNKNOWN**: {statistics.get("unknown")}
+        **TOTAL**: {statistics.get("total")}"""
+
+        analysis_table = tableToMarkdown("Analysis history", analysis_history)
+        markdown = f"{markdown}\n {analysis_table}"
+
     third_party = report_base.get("third_party_reputations")
-    third_party_statistics = third_party.get("statistics")
-    third_party_sources = third_party.get("sources")
+    if third_party:
+        third_party_statistics = third_party.get("statistics")
+        third_party_sources = third_party.get("sources")
 
-    markdown = f"""## ReversingLabs URL Threat Intelligence report for URL {url}\n **Requested URL**: {report_base.get(
-        "requested_url")}
-    **Classification**: {classification}
-    **First analysis**: {analysis.get("first_analysis")}
-    **Analysis count**: {analysis.get("analysis_count")}\n ### Last analysis\n **Analysis ID**: {last_analysis.get(
-        "analysis_id")}
-    **Analysis time**: {last_analysis.get("analysis_time")}
-    **Final URL**: {last_analysis.get("final_url")}
-    **Availability status**: {last_analysis.get("availability_status")}
-    **Domain**: {last_analysis.get("domain")}
-    **Serving IP Address**: {last_analysis.get("serving_ip_address")}\n ### Statistics\n **KNOWN**: {statistics.get(
-        "known")}
-    **SUSPICIOUS**: {statistics.get("suspicious")}
-    **MALICIOUS**: {statistics.get("malicious")}
-    **UNKNOWN**: {statistics.get("unknown")}
-    **TOTAL**: {statistics.get("total")}"""
-
-    analysis_table = tableToMarkdown("Analysis history", analysis_history)
-    markdown = f"{markdown}\n {analysis_table}"
-
-    markdown = f"""{markdown}\n ### Third party statistics\n **TOTAL**: {third_party_statistics.get("total")}
-    **MALICIOUS**: {third_party_statistics.get("malicious")}
-    **CLEAN**: {third_party_statistics.get("clean")}
-    **UNDETECTED**: {third_party_statistics.get("undetected")}\n"""
-
-    sources_table = tableToMarkdown("Third party sources", third_party_sources)
-    markdown = f"{markdown}\n {sources_table}"
+        markdown += f"""\n ### Third party statistics\n **TOTAL**: {third_party_statistics.get("total")}
+        **MALICIOUS**: {third_party_statistics.get("malicious")}
+        **CLEAN**: {third_party_statistics.get("clean")}
+        **UNDETECTED**: {third_party_statistics.get("undetected")}\n"""
+
+        sources_table = tableToMarkdown("Third party sources", third_party_sources)
+        markdown = f"{markdown}\n {sources_table}"
```

Comment:
Please update Integration README with new human readable output for this command..